minimint-api/src/module/audit.rs

```diff
@@ -0,0 +1,58 @@
+use crate::db::{Database, DatabaseKeyPrefix, DatabaseKeyPrefixConst};
+
+use std::fmt::Write;
+use std::sync::Arc;
+
+pub struct AuditItem {
+    database_key: Box<dyn DatabaseKeyPrefix>,
```

Comment:
Is the database key actually the thing we are interested in here? Should probably be a more abstract description (more to that later).

Reply:
I figured we might want to actually pull the records if we need to audit the DB based on the results, but I suppose that can be done manually so long as we output all the information required to query the DB.

Reply:
Ah, I think I misunderstood then, I thought this would be about just printing the audit item's type/source. In that case a `String` might suffice, but gotta check again.